dev/io.openliberty.data.internal_fat_nosql/test-applications/DataNoSQLApp/src/test/jakarta/data/nosql/web/DataNoSQLServlet.java

```diff
@@ -75,4 +76,14 @@ public void testBasicQuery() throws Exception {
                                      .sorted(Comparator.naturalOrder())
                                      .collect(Collectors.toList()));
     }
+
+    @Test
+    public void testGeneratedMetaModelsAvailable() {
+        //Check for _Employees is not needed as Jakarta process repositories in a way different than hibernate
+        try {
+            Class.forName("test.jakarta.data.nosql.web._Employee");
+        } catch (ClassNotFoundException e) {
+            fail("Static metamodel class _Employee (for Data) was not generated or available at runtime.");
+        }
+    }
 }
```

Comment:
You should also add a test that leverages the static metamodel class `_Employee`